lib/opensusebasetest.pm

```diff
@@ -440,7 +440,7 @@ sub wait_boot {
         handle_emergency if (match_has_tag('emergency-shell') or match_has_tag('emergency-mode'));
 
         if (!$nologin) {
-            if (get_var('DM_NEEDS_USERNAME')) {
+            if (get_var('DM_NEEDS_USERNAME') || check_var('VERSION', 'SLES-11')) {
```

Comment:
As tradition, we would set HDDVERSION=11-SP4 in test suite settings, then version will be changed to 11-SP4. So it is not safe to check 'SLES-11'.